spec/models/user_access_spec.rb

```diff
@@ -160,6 +160,31 @@
         end
       end
 
+      context "#accessible_blocks" do
+        before { Flipper.enable(:regions_prep) }
+
+        it "returns any blocks in the org an admin manages" do
+          block_1 = create(:region, :block, reparent_to: facility_group_3_1.region)
+          block_2 = create(:region, :block, reparent_to: facility_group_3_2.region)
+          _block_3 = create(:region, :block, name: "block_1", reparent_to: facility_group_1.region)
+          manager.accesses.create!(resource: organization_3)
+          expect(manager.accessible_block_regions(:manage)).to match_array([block_1, block_2])
+          viewer_all.accesses.create!(resource: organization_3)
+          expect(viewer_all.accessible_block_regions(:manage)).to match_array([])
+          expect(viewer_all.accessible_block_regions(:view_reports)).to match_array([block_1, block_2])
+        end
+
+        it "returns any blocks underneath any accessible_facility groups" do
```

Comment:
We have no first class notion of block access, so we are using the owning district (facility group) as a proxy for that.